test/helpers/fluentd.go

```diff
@@ -17,9 +17,9 @@ import (
 	"github.com/openshift/cluster-logging-operator/pkg/utils"
 )
 
-type fluentReceiverLogStore struct {
+type FluentReceiverLogStore struct {
```

Comment:
Why not keeping this private and enhance the LogStore interface by `ApplicationLogs()`. Just put `panic("Not implemented")` for the missing implementations in Elastic and Syslog